backend/internal/controllers/book.go

```diff
@@ -0,0 +1,37 @@
+package controllers
+
+import (
+	"github.com/go-chi/chi"
+	"net/http"
+
+	"github.com/TritonSE/words-alive/internal/database"
+)
+
+type BookController struct {
+	Books database.BookDatabase
+}
+
+// Fetches a list of all books sorted by title for the main page (/books)
+func (c *BookController) GetBookList(rw http.ResponseWriter, req *http.Request) {
+	books, err := c.Books.FetchBookList(req.Context())
+	if err != nil {
+		WriteResponse(rw, http.StatusInternalServerError, books)
```

Comment:
`books` here is probably going to be empty string, or something invalid. Just replace `books` with `"error"` for now.